src/kit/DocumentCard.tsx

```diff
@@ -1,4 +1,3 @@
-import { Card, Space } from 'antd';
```

Comment:
nit: this isn't strictly part of this ticket, but do you think we could move `DocumentCard` and `DocumentCards`? either into the `RichTextEditor` file or in a `RichTextEditor` folder? would prefer that everything top-level in this directory is an exported/documented hew component.